HumanResourceMachine/Controllers/HumanController.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System.Net;
+using HumanResourceMachine.Entities;
+using Microsoft.AspNetCore.Mvc;
+
+// For more information on enabling Web API for empty projects, visit https://go.microsoft.com/fwlink/?LinkID=397860
+
+namespace HumanResourceMachine.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class HumanController : ControllerBase
+    {
+        // Temporary database
+        private static List<Human> people = new List<Human>()
```

Comment:
Maybe you will use EF, DatabaseContext and real DB instead of this

Reply:
Done.

Reply:
After fixes click "Resolve conversation" and if everything alright and there is no more comments, merge branch